packages/gatsby-theme-newrelic/src/utils/page-tracking/segment.js

```diff
@@ -0,0 +1,67 @@
+import { AnalyticsBrowser } from '@segment/analytics-next';
+import warning from 'warning';
+import { getResolvedEnv, getSegmentConfig } from '../config';
+import Cookies from 'js-cookie';
+import { DEV_SEGMENT_WRITE_KEY } from '../constants';
+
+const warnAboutNoop = (section, platform) => {
+  warning(
+    section && platform,
+    "You have enabled page view tracking, but do not have page view tracking configured. This route change has not been tracked. Please configure the 'segment.section' and 'segment.platform' options in gatsby-config.js"
+  );
+};
+
+const trackPageView = ({ config, env, location, prevLocation, analytics }) => {
+  const { section, platform } = config;
+  const getCookie = (key) => Cookies.get(key)?.replace(/%22/g, '') || null;
+  const customer_user_id = getCookie('ajs_user_id');
+  const anonymousId = analytics.instance?.user().anonymousId();
+
+  if (!canSendPageView(config)) {
+    return warnAboutNoop(section, platform);
+  }
+
+  analytics.page(`${platform} page_viewed`, {
+    event_type: 'page_view',
+    path: location.pathname,
+    referrer: prevLocation?.href,
+    section,
+    platform,
```

Comment:
It looks like we are using 'platform' to denote part of the event name as well as that `platform` field in the properties. `platform` on line 29 will always be 'docs_pages' for all off our teams sites (which is confusing), so that can be hard coded. We might rename this key in the gatsby-config where we are setting what site it's coming from.